canonical-web-and-design/react-components
# Add initial cypress spec.

## Done
Add initial integration tests with cypress.

## QA
Run:
```PORT=<PORT> yarn test-cypress```

You should see two tests for Accordion pass.
